app/helpers/users_helper.rb

```diff
@@ -1,8 +1,23 @@
 module UsersHelper
-    def gravatar_for(user,options={size:80})
-        gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
-        image_size = options[:size]
-        gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?size=#{image_size}"
-        image_tag(gravatar_url, alt: user.name)
-    end
+  def gravatar_for(user,options={size:80})
+    gravatar_id = Digest::MD5::hexdigest(user.email.downcase)
+    image_size = options[:size]
+    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?size=#{image_size}"
+    image_tag(gravatar_url, alt: user.name)
+  end 
+  def helper_method(user)
```

Comment:
Change method name into a self-descriptive method name. you can use def admin(user)